fdb-record-layer-core/src/main/proto/record_metadata.proto

```diff
@@ -117,20 +117,20 @@ message DataStoreInfo {
   // This was introduced with FormatVersion.RECORD_COUNT_STATE
   optional RecordCountState record_count_state = 9;
 
-  // Setting a special store state
-  message SpecialStoreState {
+  // Setting a store lock state
+  message StoreLockState {
     enum State {
-      NONE = 1;
-      // In this state the store may not create new records or change an existing one. If set, the store - from the user's
+      NONE = 0;
```

Comment:
The protobuf guide says that this should be `UNSPECIFIED` or `UNKNOWN` (I think `UNSPECIFIED` in this case would make more sense).
https://protobuf.dev/programming-guides/proto2/#enum-default

Also, IIUC, it is worth remembering that protobuf will set it to `UNSPECIFIED` if it doesn't know about it (i.e. if it has the integer value of 3), which isn't necessary here, but means that we'll need to remember to update the `FormatVersion` when we add new values here.